pages/api/graphql.js

```diff
@@ -56,6 +56,7 @@ export default startServerAndCreateNextHandler(apolloServer, {
     const apiKey = req.headers['x-api-key']
     let session
     if (apiKey) {
+      // TODO replace email below? remove it?
       const [user] = await models.$queryRaw`
       SELECT id, name, email, "apiKeyEnabled"
```

Comment:
@ekzyis can you explain why we select `id`, `name`, and `email` here? I see that we use those fields to populate the `sessionFields` object a few lines below, but I guess I'm wondering the significance of them? Are all of them necessary? I am asking because with this PR, I am removing `email` and replacing it with `emailHash`, so obviously we can't keep `email` in this query. I can replace it with `emailHash`, if that's valuable, but I'm not sure if it is? Is this just the list of fields we make available via `me` in various parts of the server-side code? I don't see `me.email` used anywhere, so presumably this change would be safe to make.

Thoughts?

Reply:
I just selected these fields because that's how NextJS JWTs looked like by default. But only `id` and `name` are necessary.

>  Is this just the list of fields we make available via `me` in various parts of the server-side code? I don't see `me.email` used anywhere, so presumably this change would be safe to make.

Yes and I share your conclusion